src/NzbDrone.Core/ImportLists/Simkl/SimklParser.cs

```diff
@@ -30,14 +30,40 @@ public virtual IList<ImportListItemInfo> ParseResponse(ImportListResponse import
                 return series;
             }
 
-            foreach (var show in jsonResponse.Shows)
+            if (jsonResponse.Anime != null)
             {
-                series.AddIfNotNull(new ImportListItemInfo()
+                foreach (var show in jsonResponse.Anime)
                 {
-                    Title = show.Show.Title,
-                    TvdbId = int.TryParse(show.Show.Ids.Tvdb, out var tvdbId) ? tvdbId : 0,
-                    ImdbId = show.Show.Ids.Imdb
-                });
+                    var tentativeTVDBid = int.TryParse(show.Show.Ids.Tvdb, out var tvdbId) ? tvdbId : 0;
+
+                    if (tentativeTVDBid != 0 && string.Equals("tv", show.Anime_type))
+                    {
+                        series.AddIfNotNull(new ImportListItemInfo()
+                        {
+                            Title = show.Show.Title,
+                            ImdbId = show.Show.Ids.Imdb,
+                            TvdbId = tentativeTVDBid,
+                            MalId = int.TryParse(show.Show.Ids.Mal, out var malId) ? malId : 0,
+                        });
+                    }
+                    else
+                    {
+                        System.Console.WriteLine("Skipping import for " + show.Show.Title + " because it is not the first season of the show or is a movie.");
```

Comment:
This will need to use the logger, you can do construction injection like `SimklUserImport` does to inject the Logger: `Logger logger`

Reply:
If the anime is not the first season of a series in TheTVDB is it possible to add that anime if the user doesn't have the first season already?

Reply:
> If the anime is not the first season of a series in TheTVDB is it possible to add that anime if the user doesn't have the first season already?

I think it is not possible currently. Simkl API does not provide ids of prequels. However, they are planning on adding TVDB ids for the sequels in the future.